include/translated-post.php

```diff
@@ -321,44 +321,79 @@ public function current_user_can_read( $id, $context = 'view' ) {
 	 * @return WP_Post[] Array of posts.
 	 */
 	public function get_untranslated( $type, PLL_Language $untranslated_in, PLL_Language $lang, $search = '' ) {
-		$return = array();
+		global $wpdb;
 
-		// Don't order by title: see https://wordpress.org/support/topic/find-translated-post-when-10-is-not-enough.
+		// Don't order by title: see https://wordpress.org/support/topic/find-translated-post-when-10-is-not-enough
 		$args = array(
 			's'                => $search,
-			'suppress_filters' => 0, // To make the post_fields filter work.
-			'lang'             => 0, // Avoid admin language filter.
-			'numberposts'      => 20, // Limit to 20 posts.
+			'suppress_filters' => 0, // To make the post_fields filter work
+			'lang'             => 0, // Avoid admin language filter
+			'numberposts'      => 20, // Limit to 20 posts
 			'post_status'      => 'any',
 			'post_type'        => $type,
 			'tax_query'        => array(
 				array(
 					'taxonomy' => $this->tax_language,
-					'field'    => 'term_taxonomy_id', // WP 3.5+.
+					'field'    => 'term_taxonomy_id', // WP 3.5+
 					'terms'    => $lang->get_tax_prop( $this->tax_language, 'term_taxonomy_id' ),
 				),
 			),
 		);
```

Comment:
You can remove all useless args and keep only `number_posts`.

Reply:
Useless args removed in 5f1b6cf8.